csharp/Platform.Collections/Lists/IListExtensions.cs

```diff
@@ -208,7 +208,7 @@ public static void AddSkipFirst<T>(this IList<T> list, IList<T> elements, int sk
         /// <typeparam name="T"><para>The list's item type.</para><para>Тип элементов списка.</para></typeparam>
         /// <param name="left"><para>The first compared list.</para><para>Первый список для проверки.</para></param>
         /// <param name="right"><para>The second compared list.</para><para>Второй список для сравнения.</para></param>
-        /// <param name="contentEqualityComparer"></param>
+        /// <param name="contentEqualityComparer">Method to test for equality.</param>
```

Comment:
Each comment should be in both English and Russian (split using para elements).
English version can be modified like this: `Function to test two lists for their content equality.`